website/pages/blog/configuring-cq-with-localstack.md

```diff
@@ -0,0 +1,84 @@
+---
+title: CloudQuery and LocalStack
+tag: integration
+date: 2022/12/23
+description: >-
+  How to setup CloudQuery to work with LocalStack
+author: benjamin
+---
+
+import { HowToGuideHeader } from "../../components/HowToGuideHeader"
+import { Callout } from 'nextra-theme-docs'
+
+<HowToGuideHeader/>
+
+
+In this guide we will walk through how to configure CloudQuery to sync from a LocalStack instance.
+
+
+
+## Introduction to LocalStack
+
+LocalStack is a fully functional local cloud stack that enables developers to Develop and test their cloud and serverless applications offline.  It allows developers to test their cloud applications locally, without the need for an internet connection or an AWS account. This can be particularly useful for development and testing, as it allows developers to work on their applications offline and avoid incurring any charges for using AWS services.
```

Comment:
```suggestion
[LocalStack](https://localstack.cloud/) is a fully functional local cloud stack that enables developers to Develop and test their cloud and serverless applications offline. It allows developers to test their cloud applications locally, without the need for an internet connection or an AWS account. This can be particularly useful for development and testing, as it allows developers to work on their applications offline and avoid incurring any charges for using AWS services.
```